modules/bindgen/src/main/scala/Config.scala

```diff
@@ -26,6 +26,7 @@ case class Config(
     linkName: Option[LinkName],
     indentSize: IndentationSize,
     indents: Indentation,
+    bracesNotIndents: Braces,
```

Comment:
Let's rename this to `useBraces` (and `Braces` -> `UseBraces`) – the "not indents" part is inferred in case of Scala 3